<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="nb-no">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">565d5b03449aa1cc61f0b0f2a1239a62b6181aaf</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Runtime.DurableInstancing.InstanceCollisionException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">4732ea51d82ffdccbdc8ce60daa2c7cf7cf92c93</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">c0ae21cc78d748b32fd41e11aa48cc2f44722d31</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>A persistence provider throws this exception when it expects an instance to be in an uninitialized state but the instance is not in that state.</source>
          <target state="translated">Leverandør utholdenhet oppstår dette unntaket når det forventes at en forekomst i en ikke-initialisert tilstand, men forekomsten er ikke i denne tilstanden.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>A persistence provider throws this exception when it expects an instance to be in an uninitialized state but the instance is not in that state.</source>
          <target state="translated">Leverandør utholdenhet oppstår dette unntaket når det forventes at en forekomst i en ikke-initialisert tilstand, men forekomsten er ikke i denne tilstanden.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">Initialiserer en forekomst av den <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class using the error message.</source>
          <target state="translated">Initialiserer en forekomst av den <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse med feilmeldingen.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>The reason for the exception.</source>
          <target state="translated">Årsak for unntaket.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">Initialiserer en ny forekomst av den <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse serialiserte dataene.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>The information about the current exception.</source>
          <target state="translated">Informasjon om den gjeldende unntaket.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">Kontekstavhengig informasjon om kilden eller målet.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class using the error message and the inner exception information.</source>
          <target state="translated">Initialiserer en forekomst av den <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse med feilmeldingen og internt unntak-informasjon.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a result of a previous exception typically includes a reference to the previous exception in the &lt;xref:System.ServiceModel.ExceptionDetail.InnerException%2A&gt; property.</source>
          <target state="translated">Et unntak som er et resultatet av et tidligere unntak vanligvis inneholder en referanse til det tidligere unntaket i den &lt;xref:System.ServiceModel.ExceptionDetail.InnerException%2A&gt;egenskapen.&lt;/xref:System.ServiceModel.ExceptionDetail.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>The reason for the exception.</source>
          <target state="translated">Årsak for unntaket.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>The exception that caused the current exception.</source>
          <target state="translated">Unntaket som førte til den gjeldende unntaket.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class using the XName (the combination of namespace and name) of the command and the ID of the target instance against which the command was executed passed as parameters.</source>
          <target state="translated">Initialiserer en forekomst av den <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse ved hjelp av XName (kombinasjonen av navneområde og navn) for kommandoen, og IDen for målforekomsten kommandoen ble kjørt mot som ble sendt som parametere.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>The XName (the combination of namespace and name) of the command.</source>
          <target state="translated">XName (kombinasjonen av navneområde og navn) for kommandoen.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>The ID of the target instance against which the command was executed.</source>
          <target state="translated">IDen for Målforekomsten som kommandoen ble utført.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class using the XName (the combination of namespace and name) of the command, the ID of the target instance against which the command was executed, and the inner exception information.</source>
          <target state="translated">Initialiserer en forekomst av den <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse ved hjelp av XName (kombinasjonen av navneområde og navn) for kommandoen ID Målforekomsten som kommandoen ble kjørt, og informasjonen i det indre unntaket.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>The XName (the combination of namespace and name) of the command.</source>
          <target state="translated">XName (kombinasjonen av navneområde og navn) for kommandoen.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve">
          <source>The ID of the target instance against which the command was executed.</source>
          <target state="translated">IDen for Målforekomsten som kommandoen ble utført.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve">
          <source>The exception that caused the current exception.</source>
          <target state="translated">Unntaket som førte til den gjeldende unntaket.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class using the XName (the combination of namespace and name) of the command, the ID of the target instance against which the command was executed, the error message that explains the reason for the current exception, and the inner exception information passed as parameters.</source>
          <target state="translated">Initialiserer en forekomst av den <bpt id="p1">&lt;xref href="System.Runtime.DurableInstancing.InstanceCollisionException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse ved hjelp av XName (kombinasjonen av navneområde og navn) for kommandoen ID målforekomsten mot som kommandoen ble kjørt, feilmeldingen som beskriver årsaken til den gjeldende unntaket og internt unntaksinformasjon sendes som parametere.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve">
          <source>The XName (the combination of namespace and name) of the command.</source>
          <target state="translated">XName (kombinasjonen av navneområde og navn) for kommandoen.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>The ID of the target instance against which the command was executed.</source>
          <target state="translated">IDen for Målforekomsten som kommandoen ble utført.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve">
          <source>The reason for the exception.</source>
          <target state="translated">Årsak for unntaket.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>The exception that caused the current exception.</source>
          <target state="translated">Unntaket som førte til den gjeldende unntaket.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>